LQResultProducer.py

```diff
@@ -981,7 +1029,8 @@ def main():
 		scaleFactors = [Rz_uujj,Rtt_uujj,Rw_uvjj]
 		if not doLongLived :
 			MuMuOptTestCutFile = 'Results_'+version_name+'/OptLQ_uujjCuts_Smoothed_pol2cutoff.txt'
-			variableSpace = ['M_uu:25:100:1000','St_uujj:100:300:2500','M_uujj2:25:100:1000']
+			#variableSpace = ['M_uu:25:100:1000','St_uujj:100:300:2500','M_uujj2:25:100:1000']
+			variableSpace = ['M_uujj:25:500:2500','St_uujj:100:300:2500',dR_muon1muon2_jet1+':0.5:0:10']
```

Comment:
I guess this was an experiment? Please remove it and uncomment the previous line